x-pack/platform/plugins/shared/cases/public/components/observables/translations.tsx

```diff
@@ -177,3 +177,11 @@ export const DEFAULT_OBSERVABLE_TYPES_MODAL_BUTTON_ARIA_LABEL = i18n.translate(
     defaultMessage: 'Default observable types modal button',
   }
 );
+
+export const DEFAULT_OBSERVABLE_TYPES_MODAL_DESCRIPTION = i18n.translate(
+  'xpack.cases.caseView.observables.defaultObservableTypesModalDescription',
+  {
+    defaultMessage:
+      'When auto-extracting observables is on, available values from the following ECS fields are added as observables.',
```

Comment:
auto-extracting or auto-extract? I think either make sense, but wanted to confirm

Reply:
We don't have specific instructions on gerunds in copy (yet), so while either work, going with the non-gerund form is simpler. 


```suggestion
      'When auto-extract observables is on, available values from the following ECS fields are added as observables.',
```